# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-11-10 11:24-0500\n"
"PO-Revision-Date: 2024-11-10 11:23-0500\n"
"Last-Translator: Juan Sebastian <sebasmoralesd@gmail.com>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Translated-Using: django-rosetta 0.10.0\n"

#: .\apps\common\utils\contrib\auth\mixins.py:17
msgid "group_required attribute not set"
msgstr "atributo group_required no establecido"

#: .\apps\common\utils\contrib\auth\mixins.py:25
msgid "You do not have permission to view this page"
msgstr "No tienes permiso para ver esta página"

#: .\apps\common\utils\forms.py:9
msgid "Security Key"
msgstr "LLave de seguridad"

#: .\apps\common\utils\middleware.py:61
msgid "This IP is temporarily blocked due to suspicious activity."
msgstr "Esta IP está bloqueada temporalmente debido a actividad sospechosa."

#: .\apps\common\utils\middleware.py:62 .\apps\common\utils\views.py:61
msgid "Error 403"
msgstr "Error 403"

#: .\apps\common\utils\middleware.py:63
msgid "Access denied due to suspicious activity."
msgstr "Acceso denegado debido a actividad sospechosa."

#: .\apps\common\utils\models.py:21
msgid "Spanish"
msgstr "Español"

#: .\apps\common\utils\models.py:22
msgid "English"
msgstr "Ingles"

#: .\apps\common\utils\models.py:26
msgid "language"
msgstr "idioma"

#: .\apps\common\utils\models.py:35
msgid "created"
msgstr "creado"

#: .\apps\common\utils\models.py:41
msgid "updated"
msgstr "actualizado"

#: .\apps\common\utils\models.py:47
msgid "is active"
msgstr "esta activo"

#: .\apps\common\utils\models.py:52
msgid "priority"
msgstr "prioridad"

#: .\apps\common\utils\models.py:65
msgid "Attempts to obtain forbidden urls"
msgstr "Intentos de obtener URL prohibidas"

#: .\apps\common\utils\models.py:67
msgid "Multiple failed security key entry attempts"
msgstr "Múltiples intentos fallidos de ingreso de clave de seguridad"

#: .\apps\common\utils\models.py:70
msgid "is blocked"
msgstr "esta bloqueado"

#: .\apps\common\utils\models.py:71
msgid "current user IP"
msgstr "IP del usuario"

#: .\apps\common\utils\models.py:73
msgid "reason"
msgstr "razón"

#: .\apps\common\utils\models.py:75
msgid "blocked until"
msgstr "bloqueado hasta"

#: .\apps\common\utils\models.py:77
msgid "session information"
msgstr "información de la sesión"

#: .\apps\common\utils\models.py:91
msgid "requests"
msgstr "solicitudes"

#: .\apps\common\utils\models.py:119
msgid "Request"
msgstr "Solicitud"

#: .\apps\common\utils\models.py:120
msgid "Requests"
msgstr "Solicitudes"

#: .\apps\common\utils\views.py:45
msgid "Error 400"
msgstr "Error 400"

#: .\apps\common\utils\views.py:46
msgid "Bad Request"
msgstr "Solicitud incorrecta"

#: .\apps\common\utils\views.py:62
msgid "Prohibited Request"
msgstr "Solicitud prohibida"

#: .\apps\common\utils\views.py:77
msgid "Error 404"
msgstr "Error 404"

#: .\apps\common\utils\views.py:78
msgid "Page not found"
msgstr "Página no encontrada"

#: .\apps\common\utils\views.py:92
msgid "Error 500"
msgstr "Error 500"

#: .\apps\common\utils\views.py:93
msgid "Server error"
msgstr "Error del servidor"

#~ msgid "Your IP has been blocked due to too many incorrect attempts."
#~ msgstr "Su IP ha sido bloqueada debido a demasiados intentos incorrectos."

#, python-brace-format
#~ msgid "Cannot convert value {value} to {self.data_type}: {e}"
#~ msgstr "No se puede convertir el valor {value} a {self.data_type}: {e}"
